docs/core/logging.md

```diff
@@ -178,148 +224,286 @@ Powertools for AWS Lambda (Java) extends the functionality of Log4J. Below is an
+
+            // logged as a String
+            LOGGER.debug("{}", JsonConfig.get().getObjectMapper().writeValueAsString(order));
+
+            // Logged as JSON
+            LoggingUtils.logMessagesAsJson(true);
+            LOGGER.debug("{}", JsonConfig.get().getObjectMapper().writeValueAsString(order));
+            LoggingUtils.logMessagesAsJson(false);
+
+            // ...
         }
     }
     ```
 
+=== "Order.java"
+
+    ```java 
+    public class Order {
+        private String id;
+        private Date date;
+        private Double amount;
+    }
+    ```
+
+=== "Example CloudWatch Logs"
+
+    ```json hl_lines="3 9-13"
+	{
+		"level": "DEBUG",
+	  	"message": "{\"id\":\"435iuh2j3hb4\", \"date\":\"2023-12-01T14:48:59\", \"amount\":435.5}",
+		"timestamp": "2023-12-01T14:49:19.293Z",
+	  	"service": "payment",
+	}
+	{
+		"level": "DEBUG",
+	  	"message": {
+            "id": "435iuh2j3hb4", 
+            "date": "2023-12-01T14:48:59", 
+            "amount":435.5
+        },
+		"timestamp": "2023-12-01T14:49:19.312Z",
+	  	"service": "payment",
+	}
+    ```
+
+You can also achieve this more broadly for all JSON messages (see advanced configuration for [log4j](#log-messages-as-json-1) & [logback](#log-messages-as-json-2)).
+
+## Additional structured keys
+
+### Logging Lambda context information
+The following keys will also be added to your structured logs (unless [configured otherwise](#more-customization-1)):
+
+| Key                      | Type    | Example                                                                                | Description                        |
+|--------------------------|---------|----------------------------------------------------------------------------------------|------------------------------------|
+| **cold_start**           | Boolean | false                                                                                  | ColdStart value                    |
+| **function_name**        | String  | "example-PaymentFunction-1P1Z6B39FLU73"                                                | Name of the function               |
+| **function_version**     | String  | "12"                                                                                   | Version of the function            |
+| **function_memory_size** | String  | "512"                                                                                  | Memory configure for the function  |
+| **function_arn**         | String  | "arn:aws:lambda:eu-west-1:012345678910:function:example-PaymentFunction-1P1Z6B39FLU73" | ARN of the function                |
+| **function_request_id**  | String  | "899856cb-83d1-40d7-8611-9e78f15f32f4""                                                | AWS Request ID from lambda context |
+
+
+### Logging incoming event
+
+When debugging in non-production environments, you can instruct the `@Logging` annotation to log the incoming event with `logEvent` param or via `POWERTOOLS_LOGGER_LOG_EVENT` env var.
```

Comment:
Should we have some words in here (and maybe "Logging handler response") about what this means for a stream handler?

Reply:
what can we say?